test/Swashbuckle.AspNetCore.Newtonsoft.Test/SchemaGenerator/NewtonsoftSchemaGeneratorTests.cs

```diff
@@ -311,6 +328,17 @@ public void GenerateSchema_SetNullableFlag_IfInlineEnum()
         Assert.True(schema.Properties[nameof(TypeWithNullableProperties.NullableIntEnumProperty)].Nullable);
     }
 
+    [Fact]
+    public void GenerateSchema_AddNullEnumValue_IfInlineEnum()
+    {
+        var schemaRepository = new SchemaRepository();
+
+        var referenceSchema = Subject(o => o.UseInlineDefinitionsForEnums = true).GenerateSchema(typeof(TypeWithNullableProperties), schemaRepository);
+
+        var schema = schemaRepository.Schemas[referenceSchema.Reference.Id];
+        Assert.Null(schema.Properties[nameof(TypeWithNullableProperties.NullableIntEnumProperty)].Enum.Last());
```

Comment:
Why `Last()`? Looks like more should be asserted here, such as `Enum` not being `null`, as well as the number of values and what they are.